### ISSUE TITLE
Wrong indentation for XML header (<?xml version="1.0"?>)

### ISSUE THREAD

**Issue by Infocatcher** [beautify-web/js-beautify#783]
Input:
```xml
<?xml version="1.0"?>
<foo>
<bar/>
</foo>
```
Output:
```xml
<?xml version="1.0"?>
    <foo>
        <bar/>
    </foo>
```

So, there is useless indentation level after `<?xml version="1.0"?>`. Ans should be
```xml
<?xml version="1.0"?>
<foo>
    <bar/>
</foo>
```

**Comment by Infocatcher** [comment]
Was broken after these commits: https://github.com/Infocatcher/AkelPad_scripts/commit/37322e163378fc4f242fe56da6f991065f9cd0bb
(and <a href="http://akelpad.sourceforge.net/forum/viewtopic.php?p=29455#29455">thanks to Skif_off for testing</a>)

**Comment by bitwiseman** [comment]
Fixed with #683

**Comment by Infocatcher** [comment]
Thanks!
